_test/test_transformation/test_spher_proj.m

```diff
@@ -0,0 +1,32 @@
+classdef test_spher_proj<TestCase
```

Comment:
Is this meant to be included? Unrelated.

Reply:
yes, its erroneous test came from other ticket. Commented for now

Reply:
`git rm --cached` is what you want, you could clean it up more nicely, but that will remove it from the commit.